stdlib/public/SDK/AVFoundation/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 add_swift_library(swiftAVFoundation ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
   AVFoundation.swift
   TARGET_SDKS OSX IOS IOS_SIMULATOR TVOS TVOS_SIMULATOR
+  SWIFT_COMPILE_FLAGS "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}"
+  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
```

Comment:
This is a lot of boilerplate for every library, could `add_swift_library` automatically append these flags if the library is a target library?